example/per.jonprl

```diff
@@ -0,0 +1,125 @@
+Operator per-squash : (0).
+Prefix 7 "⇃" := per-squash.
+[⇃ A] =def= [per(lam(_.lam(_.A)))].
```

Comment:
I thought that the half-downward-arrow was for the "quotient squash", which expresses the PER `{(M, N) | M in A, N in A}`... This one seems different, since its inhabitants are anything at all so long as `A` is inhabited; am I reading this right?